docs/manual-EN/2.query-language/4.statement-syntax/1.data-definition-statements/TTL.md

```diff
@@ -1,79 +1,111 @@
 # TTL (time-to-live)
 
-With **TTL**, **Nebula Graph** provides the ability to filter the expired data automatically when traversing vertices and edges. The system will automatically delete the expired data during the compaction phase, without a delete operation that is explicitly issued by the console.
+With **TTL**, **Nebula Graph** provides the ability to delete the expired vertices or edges automatically. The system will automatically delete the expired data during the compaction phase. Before compaction, query will filter the expired data.
 
-`ttl_col` indicates the TTL column, while `ttl_duration` indicates the duration of the TTL. When the sum of the TTL column and the ttl_duration is less than the current time, we consider the data as expired. The `ttl_col` type is integer or timestamp, and is set in seconds. `ttl_duration` is also set in seconds.
+TTl requires `ttl_col` and `ttl_duration` together. `ttl_col` indicates the TTL column, while `ttl_duration` indicates the duration of the TTL. When the sum of the TTL column and the ttl_duration is less than the current time, we consider the data as expired. The `ttl_col` type is integer or timestamp, and is set in seconds. `ttl_duration` is also set in seconds.
 
 ## TTL configurations
 
-The time to live value is set in seconds.
+- The `ttl_duration` is set in seconds. If it is set to -1 or 0, the vertex property in the tag does not expire.
 
-- If TTL is set, when the sum of the `ttl_col` and the `ttl_duration` is less than the current time, we consider the data as expired.
+- If TTL is set, when the sum of the `ttl_col` and the `ttl_duration` is less than the current time, we consider the the vertex property in the tag as expired.
```

Comment:
the vertex properties of this tag will expire after the specified number of seconds configured by `ttl_duration` has passed since the `ttl_col` field value.